docs/release.md

```diff
@@ -20,6 +20,10 @@
    $ source venv/bin/activate
    $ pip install -r internal/buildscripts/packaging/release/requirements.txt
    ```
+1. Clone [cargo](https://cd.splunkdev.com/observability/cloudinfra/cargo/-/tree/master/)
+   repository for local access to the [okta-aws-setup](
+   https://cd.splunkdev.com/observability/cloudinfra/cargo/-/blob/master/okta-aws-setup/okta-aws-setup.sh)
+   script.
```

Comment:
Link checker will fail since these are Internal addresses.

Reply:
I've removed the internal addresses.